raft/raft.go

```diff
@@ -209,10 +186,22 @@ type Raft struct {
 	// when raft changes its state to follower or candidate.
 	randomizedElectionTimeout int
 
-	tick func()
 	step stepFunc
```

Comment:
can we include it in delete?

Reply:
We use `func (r *Raft) tick()` replace it (make room to write `// Your Code Here`), we need to delete it in case of naming conflict.

Reply:
I mean remove stepFunc